src/scripts.js

```diff
@@ -1 +1,33 @@
-console.log('Hello world');
+let recipeRepository; 
+
+const recipeCarousel = document.querySelector('.recipe-carousel')
+const searchBox = document.querySelector('.search-box');
+
+window.addEventListener('load', compileRecipeRepository)
+window.addEventListener('load', populateFeaturedRecipesCarousel);
+document.addEventListener('keydown', searchAllRecipes);
+
```

Comment:
looks good. we might want to place the event listeners at the bottom of the page to avoid issues.